data/distro.yml

```diff
@@ -208,7 +208,13 @@
     <ol class="distrotut">
       <li>
         <h2>Install Flatpak</h2>
-        <p>Flatpak can be installed on Gentoo using the Flatpak overlay. See <a href="https://github.com/fosero/flatpak-overlay">the readme for instructions</a>.</p>
+        <p>To install Flatpak, enable the ~amd64 keyword for sys-apps/flatpak, acct-user/flatpak and acct-group/flatpak:</p>
+        <pre><code>
+          <span class="unselectable">#</span> echo 'sys-apps/flatpak ~amd64\nacct-user/flatpak ~amd64\nacct-group/flatpak ~amd64\ndev-util/ostree ~amd64' >> /etc/portage/accept_keywords/flatpak
```

Comment:
This command isn't present on [the Gentoo wiki page for Flatpak](https://wiki.gentoo.org/wiki/Flatpak). Is it needed?

Reply:
Based on the [Portage tree](https://gitweb.gentoo.org/repo/gentoo.git/tree/sys-apps/flatpak/flatpak-1.9.2.ebuild#n15), it is *required* to enable the testing keyword (`~amd64`) for those packages.

Reply:
Can confirm:

```bash
TheMainDesktop / # emerge -a sys-apps/flatpak

These are the packages that would be merged, in order:

Calculating dependencies... done!

!!! All ebuilds that could satisfy "sys-apps/flatpak" have been masked.
!!! One of the following masked packages is required to complete your request:
- sys-apps/flatpak-1.9.2::gentoo (masked by: ~amd64 keyword)
- sys-apps/flatpak-1.8.2::gentoo (masked by: ~amd64 keyword)

For more information, see the MASKED PACKAGES section in the emerge
man page or refer to the Gentoo Handbook.
```